Frontend/frontend/src/Components/Chores.js

```diff
@@ -43,8 +48,9 @@ export default class Chores extends Component {
 
     render () {
         return (
-            <Container>
-                <h2>Flat Chores</h2>
+            <Container >
+                <div class='section-header' style={{"width":"45%", "marginBottom": "50px"}}>Flat Chores</div>
```

Comment:
What convention are we following around styling? We have quite a bit of CSS in the chores.css file so perhaps move this kind of code into there. Although, for the material UI chores table I had inline styling as adding it to the chores.css didn't work for some reason

Reply:
Yea I think we talked yesterday about doing inline styling only when one property is being applied so maybe this should move to the css file

Reply:
Yea, I had this issue too. When I added the styling to the css file, it didn't work. So I did some of my stylings inline